### PR TITLE
Added fallback dummy IP to Public IP discovery service

### PR DESCRIPTION
**Pull request by illuque-iov**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by Vovchyk — DISMISSED** [review]

File: rskj-core/src/main/java/org/ethereum/config/SystemProperties.java [hunk 1]
```diff
@@ -499,11 +499,17 @@ private InetAddress getMyPublicIpFromRemoteService() {
...
         InetAddress bindAddress = getBindAddress();
-        if (bindAddress.isAnyLocalAddress()) {
-            throw new RuntimeException("Wildcard on bind address it's not allowed as fallback for public IP " + bindAddress);
+        if (!bindAddress.isAnyLocalAddress()) {
+            return bindAddress;
         }
 
-        return bindAddress;
+        try {
+            // workaround to provide a last resource fallback to calculate node's public IP
```
Comment (Vovchyk) [inline rskj-core/src/main/java/org/ethereum/config/SystemProperties.java:507]: perhaps `last resort` ?

Comment (illuque-iov) [reply rskj-core/src/main/java/org/ethereum/config/SystemProperties.java:507]: Yep, literal translation here :), will fix it.

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by illuque-iov** [comment]
pipeline:run

**Review by Vovchyk — APPROVED** [review]

**Pull request merged**